src/libcore/task/wake.rs

```diff
@@ -112,10 +112,6 @@ impl LocalWaker {
     /// `UnsafeWake` trait will typically provide a wrapper that calls this
     /// but you otherwise shouldn't call it directly.
     ///
-    /// If you're working with the standard library then it's recommended to
-    /// use the `LocalWaker::from` function instead which works with the safe
-    /// `Rc` type and the safe `LocalWake` trait.
-    ///
```

Comment:
I also removed these comments, since there is no `LocalWake` trait.

Reply:
Good catch! Sorry, this got left in after I rewrote this code to work differently. I agree that this comment should go away, and it should instead suggest using the `local_waker_from_nonlocal` or `local_waker` functions in `std::task`.